File: src/main/scala/org/openapitools/server/model/LogisticPickupPointDTO.scala
package org.openapitools.server.model


/**
 * Описание пункта вывоза для возврата.
 *
 * @param id Идентификатор пункта вывоза. for example: ''null''
 * @param name Название пункта вывоза. for example: ''null''
 * @param address  for example: ''null''
 * @param instruction Дополнительные инструкции к вывозу. for example: ''null''
 * @param `type`  for example: ''null''
 * @param logisticPartnerId Идентификатор логистического партнера, к которому относится логистическая точка. for example: ''null''
*/
final case class LogisticPickupPointDTO (
  id: Option[Long] = None,
  name: Option[String] = None,
  address: Option[PickupAddressDTO] = None,
  instruction: Option[String] = None,
  `type`: Option[LogisticPointType] = None,
  logisticPartnerId: Option[Long] = None
)

File: ympa_fsharp_giraffe_server/src/model/LogisticPickupPointDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.LogisticPointType
open ympa_fsharp_giraffe_server.Model.PickupAddressDTO

module LogisticPickupPointDTO =

  //#region LogisticPickupPointDTO


  type LogisticPickupPointDTO = {
    Id : int64;
    Name : string;
    Address : PickupAddressDTO;
    Instruction : string;
    Type : LogisticPointType;
    LogisticPartnerId : int64;
  }
  //#endregion
